bin/release

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env ruby
+require 'pathname'
+require 'active_support/all'
+
+# path to your application root.
+APP_ROOT = Pathname.new File.expand_path("../../",  __FILE__)
+
+def run(command)
+  abort "command failed (#{$?}): #{command}" unless system command
+end
+
+Dir.chdir APP_ROOT do
+  puts %q[
+   _             _
+  | |           (_)
+  | | ___   __ _ _ _ __    __ _  _____   __
+  | |/ _ \ / _` | | '_ \  / _` |/ _ \ \ / /
+  | | (_) | (_| | | | | || (_| | (_) \ V /
+  |_|\___/ \__, |_|_| |_(_)__, |\___/ \_/
+            __/ |          __/ |
+           |___/          |___/
+  ]
+
+  RC_BRANCH = "stages/rc-#{3.days.from_now.strftime('%Y-%m-%d')}"
+
+  puts "==> This script will perform all of the following:"
+  puts "1. Checkout a new RC branch based on stages/int"
+  puts "2. Merge master into the RC branch"
+  puts "3. Force push the RC branch to stages/int on GitHub"
```

Comment:
Why aren't we branching the RC off of master, and then merging the RC into stages/int? That would allow us to avoid a force push.

Reply:
This was the result of the discussion we had around how to let people use the "squash and merge" feature. @tbaxter-18f did some research and concluded that in order to allow the "squash and merge" feature, and prevent merge conflicts on the `int`, `staging`, and `prod` branches when cherry picking commits, we had to branch the RC off of `int`, not `master`. The Release Management Wiki was updated a while back to reflect this, but I don't think everyone was following the new rules, which might explain why I had merge conflicts in `staging` and `prod` for RC 54 and other strange things like the diff being much larger than it should have been.